CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+
+<a name="EdgeX Command Line Interface (found in edgex-cli) Changelog"></a>
+## EdgeX Command Line Interface (CLI)
+[Github repository](https://github.com/edgexfoundry/edgex-cli)
+
+### Change Logs for EdgeX Dependencies
+- [go-mod-core-contracts](https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/CHANGELOG.md)
+
+## [v2.2.0] Kamakura - 2022-05-11  (Not Compatible with 1.x releases)
+
+### Bug Fixes 🐛
+- Replace slack.edgexfoundry.org with edgexfoundry.slack.com ([#b653c32](https://github.com/edgexfoundry/edgex-cli/commits/b653c32))
```

Comment:
I suggest moving this one to docs at it is a correction in README. IMO, the PR should have had a docs semantic prefix